free-programming-cheatsheets.md

```diff
@@ -99,6 +99,7 @@
 ### Python
 
 * [Comprehensive Python Cheatsheet](https://gto76.github.io/python-cheatsheet) - Jure Šorn (HTML)
+* [Learn Python in Y minutes](https://learnxinyminutes.com/docs/python/) - Learn X in Y Minutes Python Cheat sheet
```

Comment:
The convention is to denote the author/vendor at the end, not a description or title. 
We already know the document it's in, the section it's in the document, and the title on the left. 

```suggestion
* [Learn Python in Y minutes](https://learnxinyminutes.com/docs/python/)
```